markdown/dev/howtos/git/editing-on-github/en.md

```diff
@@ -0,0 +1,63 @@
+This 'How to' is designed to help get you started editing content using the [Github website](https://github.com/freesewing) where you can edit pattern instructions and website pages.  For info, it is also possible to make changes using the Git desktop application and/or your computer's command line; these advanced topics are beyond the scope of this 'How to'.
+
+Git allows Freesewing supporters to suggest changes and helps the administrators to review the code and implement approved changes. If you are new to Git, you will need to [create an account](https://github.com/signup?user_email=&source=form-home-signup).
+
+All pattern instructions and web pages (including this 'How to') are written using Markdown code. Please see our [Markdown guide](https://freesewing.dev/guides/markdown) for more information on this.
+
+All the *live* code and images are stored in the Freesewing repository.  For security, most contributors don't have permission to change the 'live' code.  Instead we must follow three steps to make changes:
+
+1. *Fork* (create a copy of) the repository.  This allows you to experiment with changes without affecting the original project.
+2. Make the edits.
+3. Save your edits and submit a *pull request* to let the admins know about your changes.
+
+The Github documentation provides a more [detailed explanation](https://docs.github.com/en/get-started/quickstart/fork-a-repo) of this.
+
+## Fork the repository
+- On Github.com, navigate to the freesewing [repository](https://github.com/freesewing/freesewing)
+- In the top-right corner of the page, click **Fork**
+![Fork button](fork_button.png)
+- It is helpful to give your fork a name to help you and others know what has changed. E.g. if you are going to edit the Simone pattern instructions to include info on using velcro fastenings, you could call your fork "Simone-Velcro".
+- Please also include a brief Description to help the maintainers know what you are changing. In this example, you might want to write: "Updated Simone instructions with option for velcro fastenings."
+- Click the "Create fork" button.
+![Create fork button](fork-create-button.png)
+- Create a *Branch* where you will make your edits.  Click the drop-down "Develop" menu button at the top-left of the page and give your branch a meaningful, brief name, e.g.
+![Create Branch button](create-branch.png)
+<Note>
+Double-check that the "Develop" drop-down menu displays the name of the branch you just created.
+</Note>
+
+## Make the edits
+- Navigate to the folder containing the file that you want to edit.
+<Note>
+To help you understand how the code is organised, here is the path to the Simone Instructions:
+```freesewing/markdown/org/docs/patterns/simone/instructions/
+```
+This folder contains several files, one for each language.
+![Path to Simone instructions](simone-instructions-path.png)
+</Note>
+- Select on the file you want to update. In this example, we will edit the English language instructions. 
```

Comment:
Please tell people to **only** update the English and ignore all the rest.
Because the rest is translated through translation software, rather then update the markdown files.

Also, the `markdown/dev` docs are only available in English, so it's probably best to focus on that.